client/modules/workspace/src/AppsWizard/FormField.tsx

```diff
@@ -75,10 +75,14 @@ const QueueStatus: React.FC<{
   const { getValues } = useFormContext();
   const selectedSystemId = getValues('configuration.execSystemId');
   const displayName = getSystemDisplayName(selectedSystemId);
-  const { data: queueData } = useSystemQueue(displayName);
+  const { data: systems } = useSystemOverview();
+  const selectedSystem = systems?.find(
+    (sys) => sys.display_name === displayName
+  );
```

Comment:
Can we key off of `sys.hostname` to match against our actual `selectedSystem` object? We'd have to grab the `currentExecSystem` object from `AppsSubmissionForm`, or find it via `getExecSystemFromId`. This way, we could get the system status for any system with a valid host, rather than depend on our system id being named a certain way.